test/Faker/Container/ContainerBuilderTest.php

```diff
@@ -0,0 +1,69 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Test\Container;
+
+use Faker\Container\ContainerBuilder;
+use Faker\English\FileExtension;
+use Faker\Test\TestCase;
+use Psr\Container\ContainerInterface;
+
+/**
+ * @author Tobias Nyholm <tobias.nyholm@gmail.com>
+ */
+final class ContainerBuilderTest extends TestCase
+{
+    public function testBuildEmpty()
+    {
+        $builder = new ContainerBuilder();
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuild()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(FileExtension::class);
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
+    }
+
+    public function testBuildWithDuplicates()
+    {
+        $builder = new ContainerBuilder();
+        $builder->add(FileExtension::class);
+        $builder->add(FileExtension::class);
+        $container = $builder->build();
+
+        self::assertInstanceOf(ContainerInterface::class, $container);
```

Comment:
What are we testing here?

Reply:
We are making sure that there will not be a future update to the container builder that dont allow us to replace a definition. 

Replacing definition is super important for ie `fakerphp/swedish`. I imagine an implementation of a swedish factory will look like this: 


```php
class Factory
{
    public static function sweden(): Generator
    {
        $builder = new ContainerBuilder();
        foreach (ContainerBuilder::defaultExtensions() as $id => $definition) {
            $builder->add($definition, $id);
        }

        // Add Swedish specific extension to replace some default ones
        $builder->add(SwedishPerson::class, PersonInterface::class);
        
        return $builder->build();
    }
}

```